<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitForEventWidget</class>
 <widget class="QWidget" name="WaitForEventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title">Edit event to wait for</string>
  </property>
  <property name="whatsThis">
   <string comment="@info:whatsthis">&lt;para&gt;Set the receiver name and the event to wait for.&lt;/para&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="WaitForEventlVerticalLayout">
   <item>
    <widget class="QGroupBox" name="waitForEventGroupBox">
     <property name="title">
      <string comment="@title:group">Wait for event</string>
     </property>
     <layout class="QHBoxLayout" name="waitForEventGroupBoxHorizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="labelVerticalLayout">
        <item>
         <widget class="QLabel" name="receiverNameLabel">
          <property name="text">
           <string comment="@label:textbox">Receiver name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>receiverNameLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eventNameLabel">
          <property name="text">
           <string comment="@label:textbox">Event name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>eventNameLineEdit</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="valueVerticalLayout">
        <item>
         <widget class="KLineEdit" name="receiverNameLineEdit">
          <property name="whatsThis">
           <string comment="@info:whatsthis">&lt;para&gt;The name of the QObject that receives the event.&lt;/para&gt;
&lt;para&gt;Note that the name is not the class of the object, but the string returned by its objectName() method.&lt;/para&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="eventNameLineEdit">
          <property name="whatsThis">
           <string comment="@info:whatsthis">&lt;para&gt;The name of the event.&lt;/para&gt;
&lt;para&gt;The name must not contain the &quot;QEvent::&quot; prefix, only the pure name of the event.&lt;/para&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="waitForEventWidgetSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
